server.py

```diff
@@ -33,17 +32,9 @@ def download(number):
 
 @app.route('/upload', method = 'POST')
 def file_upload():
-    upload = request.files.get('upload')
-    name, ext = os.path.splitext(upload.filename)
-
-    save_path = "./files/user_outputs/"
-    upload.save(save_path)  # appends upload.filename automatically
+    uploaded = request.files.get('upload').file.read() #uploaded outputs by user
     
-    userpath = "./files/user_outputs/" + name + '.' + ext
-
-    with open('./files/expected_outputs/expected_output.txt','r') as f2 , open(userpath,'r') as f1:
-        ans = True
-        ans = f1.read().strip() == f2.read().strip()
+    ans = (uploaded==expected)
```

Comment:
since it's string comparison it might be a good idea to use `strip()` to avoid having mismatches due to extra `\n` in the end of any one file.

Reply:
Yeah. I'll do it straight away.

Reply:
Nope. It has to be done for uploaded as well. I have made the required changes.